components/blitz/resources/omero/api/JobHandle.ice

```diff
@@ -18,21 +18,101 @@ module omero {
     module api {
 
         /**
-         * See <a href="http://downloads.openmicroscopy.org/latest/omero5.2/api/ome/api/JobHandle.html">JobHandle.html</a>
+         * Allows submission of asynchronous jobs.
+         * <p>
+         * NOTE: The calling order for the service is as follows:
+         * <ol>
+         * <li>{@link #submit} <em>or</em> {@link #attach}</li>
+         * <li>any of the other methods</li>
+         * <li>{@link #close}</li>
+         * </ol>
+         * </p>
+         * Calling {@link #close} does not cancel or otherwise change
+         * the Job state. See {@link #cancelJob}.
          **/
         ["ami", "amd"] interface JobHandle extends StatefulServiceInterface
             {
+                /**
+                 * Submits a {@link omero.model.Job} and returns its database
+                 * id. The only fields directly on status which are editable
+                 * are <em>message</em>, <em>scheduledFor</em> and
+                 * <em>status</em>. The latter two must be sensible.
+                 *
+                 * @param job Not null
+                 */
                 long submit(omero::model::Job j) throws ServerError;
+
+                /**
+                 * Returns the current {@link omero.model.JobStatus} for the
+                 * Job id.
+                 *
+                 * @throws ApiUsageException
+                 *             if the {@link Job id} does not exist.
+                 */
                 omero::model::JobStatus attach(long jobId) throws ServerError;
+
+                /**
+                 * Returns the current {@link omero.model.Job}
+                 */
                 idempotent omero::model::Job getJob()  throws ServerError;
+
+                /**
+                 * Returns the current {@link omero.model.JobStatus}. Will
+                 * never return null.
+                 */
                 idempotent omero::model::JobStatus jobStatus()  throws ServerError;
+
+                /**
+                 * Returns <code>null</code> if the {@link omero.model.Job} is
+                 * not finished, otherwise the {@link omero.RTime} for when it
+                 * completed.
+                 */
                 idempotent omero::RTime jobFinished()  throws ServerError;
+
+                /**
+                 * Returns the current message for job. May be set during
+                 * processing.
+                 */
                 idempotent string jobMessage()  throws ServerError;
+
+                /**
+                 * Returns <code>true</code> if the {@link omero.model.Job} is
+                 * running, i.e. has an attached {@link omero.model.Process}.
```

Comment:
The i.e. here has the effect of the index entry being:

`Returns true if the model::Job is running, i.e.`

although out with this PR a change to

```
     * Returns <code>true</code> if the {@link omero.model.Job} is
     * running. I.e. has an attached {@link omero.model.Process}.
```

would make for a neater index entry.